autoload/neomake/utils.vim

```diff
@@ -129,9 +129,12 @@ function! neomake#utils#MakerFromCommand(shell, command) abort
             \shell_name) >= 0
         let args = ['-c', command]
     else
-        " TODO Windows support (at least)
-        throw "Shell not recognized; can't build command"
+        let shell_name = split(a:shell, '\\')[-1]
+        if (shell_name == 'cmd.exe')
+            let args = ['/C', command]
```

Comment:
&shellcmdflag instead of '/C', you can probably do it for any shell, not just cmd.exe